sql/handler.h

```diff
@@ -3333,6 +3356,8 @@ class handler :public Sql_alloc
     return false;
   }
   int get_lock_type() const { return m_lock_type; }
+  virtual bool update_field_defs_with_zip_dict_info()
```

Comment:
Please add a header comment, also the sole caller does not check the return value, should it be checked or should it be void?